### PR TITLE
Visitor can see collection of articles on mobile application

### PR DESCRIPTION
**Pull request by elvitak**
### This PR contains:

- add react navigation/native, react navigation/native-stack, axios, react-native-safe-area-context, react-native-screens
- add fixtures file
-  display articles
- remove setting fixtures folder false in cypress.json 
- add spec for checking title of first child inside article collection
- cleanup view wrappers

https://www.pivotaltracker.com/story/show/181033696

@Mljungho @DefyCab @giacoletti

### CODE REVIEW

File: components/ArticlesList.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+const ArticlesList = ({ navigation }) => {
+  const [articles, setArticles] = useState([]);
+
+  const fetchArticles = async () => {
+    try {
+      const response = await axios.get(
+        "https://yesterdays-news-api.herokuapp.com/api/articles"
+      );
+      setArticles(response.data.articles);
+      debugger;
```
Comment (giacoletti) [inline components/ArticlesList.js]: debugger

File: components/ArticlesList.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    } catch (error) {}
```
Comment (giacoletti) [inline components/ArticlesList.js]: You can display a message on the view if we get an error from the api call, kinda like how we do it on the client application

**Review by giacoletti — CHANGES_REQUESTED** [review]

File: components/ArticlesList.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import {
+  StyleSheet,
+  FlatList,
+  Text,
+  View,
+  TouchableOpacity,
```
Comment (giacoletti) [inline components/ArticlesList.js]: this component has not been used

File: components/ArticlesList.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+} from "react-native";
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { shouldUseActivityState } from "react-native-screens";
```
Comment (giacoletti) [inline components/ArticlesList.js]: this component has not been used

**Review by giacoletti — CHANGES_REQUESTED** [review]

File: components/ArticlesList.js (hunk 1 above)
Comment (DefyCab) [reply components/ArticlesList.js]: They are great though!

File: components/ArticlesList.js (hunk 4 above)
Comment (DefyCab) [reply components/ArticlesList.js]: Gone!

File: components/ArticlesList.js (hunk 3 above)
Comment (DefyCab) [reply components/ArticlesList.js]: Gone!

File: App.js [hunk 5]
```diff
@@ -1,17 +1,16 @@
...
+const Stack = createNativeStackNavigator();
 
 const App = () => {
   return (
-    <View style={styles.container}>
-      <Text testID="header">Hello world</Text>
-    </View>
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen name="home" component={MainView} />
```
Comment (giacoletti) [inline App.js]: `<Stack.Navigator>
      <Stack.Screen
        name="Home"
        component={HomeScreen}
        options={{ title: 'My home' }}
      />
    </Stack.Navigator>`
   I would suggest a title with the name of the newspaper, Yesterdays News
   https://reactnavigation.org/docs/headers/

**Review by giacoletti — CHANGES_REQUESTED** [review]

File: App.js (hunk 5 above)
Comment (giacoletti) [reply App.js]: `<Stack.Screen name="home" component={MainView} options={{ title: 'Yesterdays News' }} />`
like this

**Review by giacoletti — CHANGES_REQUESTED** [review]

**Pull request merged**